fpdf/adapters/table_pandas.py

```diff
@@ -0,0 +1,38 @@
+from pandas import MultiIndex
+from fpdf import FPDF
+
+
+class FPDF_pandas(FPDF):
```

Comment:
Given that this adapter only adds a single method, I think we should provide a _mixin_ instead, so that `fpdf2` users can combine several _mixins_ if they want to!
```
class PandasMixin:
    def dataframe(self, df, **kwargs):
```
And that would be the end user usage:
```
from fpdf import FPDF
from fpdf.pandas import PandasMixin

class MyPDF(FPDF, PandasMixin):
    pass

pdf = MyPDF()
pdf.add_page()
pdf.set_font("Times", size=10)
pdf.dataframe(df, ...)
```